readthedocs/api/v3/serializers.py

```diff
@@ -236,7 +236,12 @@ def get_downloads(self, obj):
         data = {}
 
         for k, v in downloads.items():
-            if k in ('htmlzip', 'pdf', 'epub'):
+            if k in ('html', 'pdf', 'epub'):
+
+                # Keep backward compatibility
+                if k == 'html':
+                    k = 'htmlzip'
```

Comment:
Not sure I understand this logic? Why not just keep `htmlzip` in the dict?

Reply:
The dict returned by `version.get_downloads()` has `html` instead of `htmlzip`. I can't change the returned value of it because it's used in other places as well.

The API response used `htmlzip` and I think we need to keep the same behavior.